07_ansible_operator/hosts

```diff
@@ -0,0 +1,5 @@
+[nextcloud]
+nextcloud ansible_host=188.166.57.84 
```

Comment:
do not commit IP :)

Reply:
Fixed, entire file is generated, so I've removed it. good catch.